README.md

```diff
@@ -10,6 +10,10 @@ nameless
 PaC  
 Flashy Moves
 
+## Support Staff
+
```

Comment:
```suggestion
```

Reply:
Add a space to all or remove spaces to this one to remain consistent

Reply:
We need to add a space to all

Reply:
Then add a space to all

Reply:
Out of scope

Reply:
ok then